catalyst-gateway/bin/src/service/api/cardano/cip36/filter.rs

```diff
@@ -444,29 +443,33 @@ pub async fn get_all_registrations(
             continue;
         };
 
+        let payment_address = Cip19ShelleyAddress::try_from(row.payment_address).ok();
```

Comment:
The reality is these conversions SHOULD be `MaybeUnset` a it is in the insert.
It has two states, `unset` is valid, there is no payment_address.
Or its a valid payment address.
If its an invalid payment address, then thats an indexing error, we should log it as an `error!` and not return its data. (Ignore it because we don't know what going on if we index badly).  make sure any logged errors are able to let us try and find what was inserted into the DB wrongly.
They are either None already in the DB or a valid value